packages/wonder-blocks-timing/docs.md

```diff
@@ -0,0 +1 @@
+Abstractions for common timing APIs like `setTimeout`, `setInterval` and `requestAnimationFrame`.
```

Comment:
Docs will be fleshed out once the actual package is finished and there are exported components.